### ISSUE TITLE
Download size of dependencies in package.json

### ISSUE THREAD

**Issue by mubaraqwahab** [arve0/download-size#9]
Hello @arve0, thank you for the package!

It would be nice to have a feature like this:

```shell
download-size -p package.json
```

and get an output like:

```shell
"package.json" dependencies:
parcel-bundler@1.12.4: 14.19 MiB
postcss-modules@3.2.2: 640.74 KiB

"package.json" devDependencies:
stylelint-config-recommended@3.0.0: 2.64 KiB

Total size: 15.01 MiB
```

With multiple package.json's:

```shell
download-size -p package.json -p dir/package.json
```

Output:

```shell
"package.json" dependencies:
parcel-bundler@1.12.4: 14.19 MiB
postcss-modules@3.2.2: 640.74 KiB

"package.json" devDependencies:
stylelint-config-recommended@3.0.0: 2.64 KiB

"dir/package.json" dependencies:
react@16.13.1: 93.23 KiB
react-dom@16.13.1: 794.74 KiB

"dir/package.json" devDependencies:
prettier@2.1.1: 2.05 MiB
eslint@7.8.0: 2.49 MiB

Total size: 5.92 MiB
```

And, with other packages:

```shell
download-size react -p package.json react-dom
```

Output:

```shell
react@16.13.1: 93.23 KiB

"package.json" dependencies:
parcel-bundler@1.12.4: 14.19 MiB
postcss-modules@3.2.2: 640.74 KiB

"package.json" devDependencies:
stylelint-config-recommended@3.0.0: 2.64 KiB

react-dom@16.13.1: 794.74 KiB

Total size: 16 MiB
```

Can I attempt this? (I'm new to open source)

Thank you.

**Comment by arve0** [comment]
Hi 👋 Sounds like a great idea.

> Can I attempt this? (I'm new to open source)

Sure, give it a try 🙂

**Comment by mubaraqwahab** [comment]
Alright thank you!

**Comment by arve0** [comment]
Resolved in `v1.4.0`:

```
$ download-size webpack rollup -f package.json
package.json (npm-download-size@1.0.0):
  dependencies:
    si-prefix@0.1.0: 6.47 KiB
Error: No matching version found for v-autocomplete@1.9.0, possible versions are 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 1.0.1, 1.0.2, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.2.0, 1.3.0, 1.4.0, 1.4.1, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.5.4, 1.6.0, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.8.0, 1.8.1, 1.8.2.
    vue@2.6.12: 825.02 KiB
    vue-class-component@6.3.2: 10.33 KiB
    vue-property-decorator@7.3.0: 14.90 KiB
  devDependencies:
    @vue/cli-service@3.12.1: 15.62 MiB
    typescript@3.9.7: 9.14 MiB
    vue-template-compiler@2.6.12: 158.12 KiB
All dependencies: 25.75 MiB

rollup@2.26.11: 684.59 KiB
webpack@4.44.1: 4.03 MiB

All packages: 30.44 MiB
```
